### DISCUSSION TITLE
Increase Grid container

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6005] [Help] [answered]
Hi please how do I increase the height of the grid container, currently I have a vertical scrollbar for just 5 rows, I want to increase the grid height to take more rows before the scroll bar shows
![gridrow](https://user-images.githubusercontent.com/50826372/135612936-b32b8a6d-c826-44a9-89b4-35ed20b7e38c.PNG)

**Comment by luongnv6** [comment, marked as answer]
add this into contructor of dialog
this.form.Budget.element.css('height', '300px');

**Reply by ghost** [reply]
Can  anyone provide the detailed instructions?

**Reply by rhemasystems** [reply]
The below is the code I placed in the constructor of my dialog 
`constructor() {
            super();
            this.form.BudgetLines.element.css('height', '460px');
        }`
